Taskfile.yml

```
@@ -0,0 +1,145 @@
+
+  schema-check:
+    dir: "{{ .ROOT_DIR }} "
+
+    desc: Check Models and Serializers for schema compliance with OpenAPI .0 Standards and fail on warnings
+    cmds:
+      - doppler run --  python3 run/manage.py spectacular  --file schema/schema$(date +"%Y-%m-%d").yaml --validate
+    internal: true
+
+  schema:
+    deps: [schema-check]
+    dir: "{{ .ROOT_DIR }}" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)